install_config/persistent_storage/dynamically_provisioning_pvs.adoc

```diff
@@ -113,6 +113,79 @@ value, triggers provisioning using the single implied
 link:#enabling-provisioner-plugins[provisioner plug-in per cloud].
 ====
 
+
+[[storage-provisioner-labels]]
+=== Storage-Provisioner Labels
+
+For dynamically provisioned storage, OpenShift defines three key/value pairs,
+called _storage-provisioner labels_ in this document,
+and associates them with the storage, in a storage-specific way.
+These are normally not visible to OpenShift users.
+Their main benefit is to enable storage administrators to recognize volumes
+associated with OpenShift.
+
+The term _storage-provisioner label_ is what this documentation uses.
+The actual term that a storage administrator will understand is listed
+in the following table:
```

Comment:
"The term _storage-provisioner label_ is what this documentation uses." Probably don't need to mention this here and above. Maybe get rid of it above, and keep it here, as we're about to talk about the table?

"This document uses the term _storage-provisioner label_ in place of the three key/value pairs."

Also, is "understand" the right word? Maybe something like "use"?